makefile.shade

```diff
@@ -203,6 +204,32 @@ var buildTarget = "compile"
         }
     }
 
+#cache-coherence
+    @{
+        var coherenceCacheDir = Path.Combine(BASE_DIR, Environment.GetEnvironmentVariable("COHERENCE_CACHE_DIR") ?? ".coherence-cache");
+        
+        if (!Directory.Exists(coherenceCacheDir))
```

Comment:
I could revert this. I had it like this to test faster locally. What do you think?